### ISSUE TITLE
[request] potential hotfix release?

### ISSUE THREAD

**Issue by Nerenye** [FrSkyRC/ETHOS-Feedback-Community#1826]
Since last 2 Version had a couple of worrying bugs and lockups i would like to ask if we could have a somewhat in between release with only those major issues fixed. 

Like but not only:
#1825 
#1819 
#1823 
#1818 
#1800 
#1804 
#1816

**Comment by bsongis-frsky** [comment]
No, we will have more people unhappy

**Comment by bsongis-frsky** [comment]
Many of the issues pasted above have not been reported in previous release, but it doesn't mean they don't exist!

**Comment by Nerenye** [comment]
@bsongis-frsky 
We´ll you´re making people unhappy with unstable builds. I´m not comfortable with upgrading to 1.3 or 1.2.11 and i don´t think i´m alone with this.
Besides there are plenty of users not following with this repo and are then confronted with lockups on x12s or Emergency mode in x20 and stuff. 
I don´t feel like that's good for your reputation. 
But Ok, your decision anyways.

**Comment by m0p3d** [comment]
Agree quality of releases decreasing - e.g sensor conflict problem+ no documentation what's ongoing/was exactly implemented and (not) fixed - see release notes 1.3 and where it came from ( first seen in 1.2.11)

**Comment by bsongis-frsky** [comment]
@Nerenye @m0p3d if you know somebody (or a company) who is able to write such a piece of firmware without any bug, please tell me, I am very interested. 

We try to do our possible to avoid bugs, all commits are peer reviewed, we have non regression tests running on each pull request etc.

And you cannot say we are not reactive when a bug is discovered. 

All the bugs that are reported here on X12S are impossible to reproduce, Mike has tried, the test team has tried, and I have tried myself.

**Comment by Nerenye** [comment]
@bsongis-frsky the bugs are not the problem. i´m a programmer for myself and everyone produces bugs.
the problem is you are releasing releases with known bugs that are passibly making your plane crash or render your transmitter partially useless. And thats the same as a few years ago with the servo movement bug you had in Protokoll version one( i know thats RF not OS related but it has the same impact on your reputation) 

TLDR: it´s not about bugs and glitches its about your priorisation of these issues.
If someone reports a 3 second no input bug there should be no release until this is fixed or worked around. and i bet everybody in this repo is going tho help find it.
